### ISSUE TITLE
Request textDocument/semanticTokens/full failed.

### ISSUE THREAD

**Issue by frankhale** [denoland/vscode_deno#1026]
**Describe the bug**

Lately I've started to notice that the Deno language server is having trouble when I edit code. I am working with Deno FFI and am seeing this when I add code to my `Deno.dlopen` function. 

This displays sporadically when I am editing code:

```
[Error - 3:43:17 PM] Request textDocument/semanticTokens/full failed.
  Message: Internal error
  Code: -32603 
```

Here is the output from the Deno language server:

```
Starting Deno language server...
  version: 1.39.2 (release, x86_64-pc-windows-msvc)
  executable: C:\Users\frank\deno\deno.EXE
Connected to "Visual Studio Code" 1.85.1
Enabling import suggestions for: https://deno.land
Server ready.
unexpected positions
specifier: file:///C:/Users/frank/Documents/DEV/game-dev/deno-ffi-sandbox/deno-test/library-test.ts
open: true
start_pos: Position { line: 64, character: 98 }
end_pos: Position { line: 65, character: 2 }
[Error - 3:43:17 PM] Request textDocument/semanticTokens/full failed.
  Message: Internal error
  Code: -32603 
```

**To Reproduce**

No idea how to reproduce it, it's happening sporadically and without any warning. It happens randomly when I am editing TypeScript code. 

**Expected behavior**

No popup messages about Deno language server errors

![image](https://github.com/denoland/vscode_deno/assets/190154/b315e822-600b-4b26-bd77-19293df4f9bb)

Versions:

deno 1.39.2 (release, x86_64-pc-windows-msvc)
v8 12.0.267.8
typescript 5.3.3

![image](https://github.com/denoland/vscode_deno/assets/190154/a7568938-d8c0-461f-bfd7-16fb98264d9f)

![image](https://github.com/denoland/vscode_deno/assets/190154/8874ed88-16f9-4b2e-aa16-19d51ec27e3b)

**Comment by nayeemrmn** [comment]
@frankhale This may have been fixed in Deno v1.39.3, are you still experiencing this?

**Comment by frankhale** [comment]
I'm going to close this since it appears to not be a problem anymore. THANK YOU!!!
